### DISCUSSION TITLE
Why is Read your browsing history a required permission for Dark Reader

### DISCUSSION THREAD

**Discussion by R3ubenM** [darkreader/darkreader#12157] [Q&A]
Hi,

I was looking at permissions of extensions and i can see that Dark reader has the ability to read your browsing history.

![image](https://github.com/darkreader/darkreader/assets/156815387/62ff57be-5d2b-433a-9426-48e68b9eb68d)

Could anyone explain why this is needed?

**Comment by alexanderby** [comment]
The extension doesn't read your browsing history. But the extensions needs to check if you disabled it for a particular URL or apply site-specific fixes.

**Reply by R3ubenM** [reply]
Sorry i don't quite understand why Dark reader would need to read browser history to check if you disabled it?

Could you also clarify what "it" is so there is no confusion

**Reply by alexanderby** [reply]
"It" means the extension. You can disable it for particular websites. Please see the FAQ https://darkreader.org/help/en/#faq
